packages/website/docs/DocSearch-program.md

```diff
@@ -78,6 +78,18 @@ The information of the initially applied project is used everywhere when we depl
 
 This allow us to easily scope issues when reaching out for support.
 
+## Support
+
+We do not provide Algolia support for DocSearch, but for the products that are used by the program.
```

Comment:
I'm not sure the sentence is clear. 
Maybe something like `We do not provide implementation support for DocSearch`

Reply:
Make sense!